documentation/src/main/asciidoc/topics/properties/hotrod-client-remote.properties

```diff
@@ -0,0 +1,6 @@
+infinispan.client.hotrod.server_list=localhost:11222
+infinispan.client.hotrod.auth_realm=default
+infinispan.client.hotrod.auth_username=admin
+infinispan.client.hotrod.auth_password=changeme
+infinispan.client.hotrod.use_auth=true
+infinispan.client.hotrod.sasl_mechanism=DIGEST-MD5
```

Comment:
I think we should drop all the default properties and show only `.auth_username` and `.auth_password`

We could also point to https://infinispan.org/docs/stable/titles/hotrod_java/hotrod_java.html#configuring-hotrod-java-clients_hotrod-client-configuration in the procedure.